fl4health/model_bases/masked_layers.py

```diff
@@ -99,7 +96,40 @@ def from_pretrained(cls, linear_module: nn.Linear) -> "MaskedLinear":
 
 class MaskedConv1d(nn.Conv1d):
     """
-    Implementation of masked Conv1d layers in a manner that is analogous to MaskedLinear.
+    Implementation of masked Conv1d layers.
```

Comment:
We've been putting class documentation in the init files. I'm not sure if that's a good practice or not, but it's where we've been doing it. Do you think we should start doing "class" level docs instead?

Reply:
I've seen both practices and I'm not sure if one way is more standard than the other. I'm moving this to `__init__` for consistency for now.